docs/guidelines/content-guidelines/01-user-docs.md

```diff
@@ -22,9 +22,30 @@ To publish a document located in the `/kyma` repository, follow these steps:
 
 ## Publish a document from an existing module repositiory
 
-To publish a document located in an existing module repositiory, follow these steps:
+To publish a document located in an existing module repositiory, follow these steps.
 
 1. Create a pull request adding your content to a Markdown file located in the `/docs/user` folder in your module repository.
-2. Add a new `_sidebar.md` file including a link to your document, or update an existing `_siderbar.md` to include it.
+2. In the `docs/user` folder, add a new `_sidebar.md` file including a link to your document, or update the existing `_siderbar.md` to include it.
+
+## Publish a document from a new module repostory
+
+To publish a document located in a new module repositiory, follow the steps from [Publish a document from an existing module](#publish-a-document-from-an-existing-module-repositiory). Once completed do the following:
+
+1. In the `/kyma` repository, add a link to your module documentation main page under **Modules** in the [`/kyma/docs/_sidebar.md`](https://github.com/kyma-project/kyma/blob/main/docs/_sidebar.md) file.
```

Comment:
```suggestion
1. In the `/kyma` repository, in the [`/kyma/docs/_sidebar.md`](https://github.com/kyma-project/kyma/blob/main/docs/_sidebar.md) file, under **Modules**, add a link to your module documentation main page.
```

I'd specify the exact location first and then the instruction. However, it sounds a bit inelegant, so maybe rather:

```suggestion
1. In the `/kyma` repository, open the [`/kyma/docs/_sidebar.md`](https://github.com/kyma-project/kyma/blob/main/docs/_sidebar.md) file and under **Modules**, add a link to your module documentation main page.
```